common/api-review/app-exp.api.md

```diff
@@ -35,11 +33,57 @@ export const _DEFAULT_ENTRY_NAME = "[DEFAULT]";
 // @public
 export function deleteApp(app: FirebaseApp): Promise<void>;
 
-export { FirebaseApp }
-
-export { FirebaseAppConfig }
+// @public
+export interface FirebaseApp {
+    automaticDataCollectionEnabled: boolean;
+    readonly name: string;
+    readonly options: FirebaseOptions;
+}
+
+// @public (undocumented)
+export interface FirebaseAppConfig {
+    // (undocumented)
+    automaticDataCollectionEnabled?: boolean;
+    // (undocumented)
+    name?: string;
+}
+
+// @public (undocumented)

```

Comment:
I guess this has to be public so app-compat is able to import it from app-exp? Will it end up in the generated documentation?

Reply:
Yes, it should have a `@internal` annotation so it doesn't show up in the documentation.
Fixed.